apps/documentation/src/routes/documentation/cli.mdx

```diff
@@ -21,6 +19,22 @@ Tuono is the CLI that provides all the needed commands to handle the full-stack
 
 To list all the available commands, run `tuono -h`.
 
+```bash
+The react/rust fullstack framework
+
+Usage: tuono <COMMAND>
+
+Commands:
+  dev    Start the development environment
+  build  Build the production assets
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>It seems like there are too many consecutive spaces here. (CONSECUTIVE_SPACES[1])
Suggestions: ` `
Rule: https://community.languagetool.org/rule/show/CONSECUTIVE_SPACES?lang=en-US&subId=1
Category: TYPOGRAPHY
<!-- __reviewdog__:ChA3MmYxNTI3OGQwMjc2ZGY5EgxMYW5ndWFnZVRvb2w= -->